src/roboat_ros_node.py

```diff
@@ -18,9 +17,12 @@
 from visualization_msgs.msg import Marker, MarkerArray
 from social_vrnn.msg import lmpcc_obstacle_array as LMPCC_Obstacle_Array, svrnn_path as SVRNN_Path, svrnn_path_array as SVRNN_Path_Array
 
+sys.path.remove("/opt/ros/kinetic/lib/python2.7/dist-packages")
+import cv2
+
 PACKAGE_NAME = 'social_vrnn'
-MAX_QUERY_AGENTS = 10
-SWITCH_AXES = True
+MAX_QUERY_AGENTS = 100
```

Comment:
More query agents, because the id/keys's of the dataset can be a lot higher than 10.